kubernetes/core-secret.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: core-secret
+type: Opaque
+data:
+    token: token_placeholder
```

Comment:
I know you didn't meet `sed` bash tool so far, but the text string 'token_placeholder' is tough to read by machine. 

Better way is to set it like `<TOKEN_PLACEHOLDER>` or even better `%TOKEN_PLACEHOLDER%` as I mentioned in #45 (https://learnk8s.io/templating-yaml-with-code#using-templates-with-search-and-replace)

It helps the machine to parse the file and don't replace occasional wrong text string for example in metadata anytime.

Reply:
Hm, I have just realized that I can only add encoded values here. If I add <TOKEN> or %TOKEN% it simply does not allow me to create this secret. But if I encode this value lt actually works fine. I can for example add encoded value here and then decode it inside our shell script to use with sed command but I'm not sure if this is optimal solution.

Reply:
I will check documentation, maybe we can use some other type of secret

Reply:
Great catch! You rocks! 👌🏽 🚀 

The best solution is to have a 'final' folder that will contain final generated yaml manifests for kubectl and this file will be only as a template one. 

What do you think about it? Is it meaningful and feasible?

Reply:
Hi Jan! Thanks for the explanation! Yes, that sounds good and reasonable. As I can see you have already changed value to <CORE_TOKEN>. Can we now push this PR?

Reply:
Thanks a lot for reminder. 👍🏽 Of course, we can merge it. I'll just do it. 👌🏽